.travis.yml

```diff
@@ -0,0 +1,55 @@
+sudo: required
+
+language: go
+
+services:
+  - docker
+
+addons:
+  apt:
+    packages:
+      - docker-ce
+
+os:
+  - linux
+
+go:
+  - "1.12"
+
+env:
+  global:
+    - BUILD_VERSION=$(echo ${TRAVIS_COMMIT} | cut -c 1-10)
+    - MAIN_GO_VERSION=1.12
+    - GORACE="halt_on_error=1"
+
+install:
+  - make deps
+
+script:
+  - make test;
+  - if [[ "$TRAVIS_GO_VERSION" == "$MAIN_GO_VERSION" && "$TRAVIS_OS_NAME" == "linux" ]]; then
+    release_latest=$( [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_TAG" == "" && "$TRAVIS_PULL_REQUEST_BRANCH" == "" ]] && echo "yes" || echo "no" );
+    release_tag=$( [[ "$TRAVIS_TAG" != "" ]] && echo "yes" || echo "no" );
+
+    if [[ $release_latest == "yes" ]]; then
+    docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
+    BUILD_VERSION="latest" make docker
```

Comment:
req: this won't work as expected as you are setting the `v` prefix in https://github.com/iov-one/chatbot/blob/master/Makefile#L3

Reply:
I have also noticed that you push to `iov1` repo on docker hub. I think `iov1ops` is a better fit as this is not any of our products.

Reply:
Yeah, it makes sense I guess, will fix that